app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -46,24 +45,24 @@ open class FenixApplication : Application() {
     }
 
     open fun setupApplication() {
-        // loadExperiments does things that run in parallel with the rest of setup.
-        // Call the function as early as possible so there's maximum overlap.
-        experimentLoader = loadExperiments()
-
+        setupCrashReporting()
```

Comment:
I was wondering whether `setupLogging` should happen before `setupCrashReporting` since the crash reporter does also log. But thinking more about it I think this change is fine: It will not log from init - only later and then it's setup - and it's good to catch crashes in the log setup code.